task.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# создаём каталог task с вложенными директориями
+# task
+#   dir1
+#   dir2
+#   dir3
+#       dir4
+mkdir -p /task/dir1 /task/dir2 /task/dir3 /task/dir3/dir4
```

Comment:
Можно использовать `./`, но не `/`, так как это уже идет от корня системы.
Так же при использовании флага `-p` можно опускать промежуточные папки
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```

Reply:
Упустила это, спасибо!